src/api/article/article.model.tests.js

```diff
@@ -35,13 +38,29 @@ describe('Article model', () => {
     expect(errorHandler).to.have.been.called();
   });
 
-  it('should fail to save video article | no video', async () => {
+  it('should fail to save article | no images', async () => {
+    const errorHandler = spy(({ message }) => {
+      expect(message).to.not.empty();
+      expect(message.images.type).to.includes('required');
+    });
+
+    await Article({ ...fiberyIds, type: 'text', metadata })
+      .save()
+      .catch(errorHandler);
+
+    expect(errorHandler).to.have.been.called();
+  });
+
+  // for now `no video` -> `type !== 'video'` 🤷
```

Comment:
what does the emoji mean? you don't know why's that happening or you're unsure if that's fine?

Reply:
@UladBohdan, since the type is now calculated property - it's a bit strange

Reply:
probably we don't need to store type explicitly? if `video` property is not empty - then it is a video-type, if `audio` - then audio-type, if none - regular article. We can add type on population